composer.json

```diff
@@ -6,6 +6,8 @@
     "replace": {
         "ezsystems/ezplatform-query-fieldtype": "*"
     },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
Can we follow one pattern when dealing with indirect dev dependencies? ATM we're adding them to require-dev

Reply:
You are probably right. There are side-effects of using "prefer-stable" option after all.

Fixed in 3ccfdb6c.